### PR TITLE
Create Integration-Samanage.yml

### PR DESCRIPTION
**Pull request by shadejinx**
This is the first version of my Samanage integration. Currently it only exposes hardware and user queries. I will be adding case query and modification in the next version.

### CODE REVIEW

File: Integrations/Integration-Samanage.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    - name: param
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - name
+      - email
+    - name: query
+      required: true
+    description: Query Samanage for user information
+  dockerimage: aviadlich/python-requests
```
Comment (meirwah) [inline Integrations/Integration-Samanage.yml]: you can switch this to demisto/python
Our official image includes python requests

File: Integrations/Integration-Samanage.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+commonfields:
+  id: Samanage
+  version: -1
+name: Samanage
+display: Samanage
+category: Data Enrichment
```
Comment (meirwah) [inline Integrations/Integration-Samanage.yml]: Is it really Data Enrichment ? isn't it  "IT Services" or "Case Management"?

Comment (shadejinx) [inline Integrations/Integration-Samanage.yml]: Probably. My current use case is data enrichment, but I have plans to add minimal case management in the future.

File: Integrations/Integration-Samanage.yml (hunk 1 above)
Comment (shadejinx) [inline Integrations/Integration-Samanage.yml]: excellent, I will update my scripts to include.

**Comment by gshriki** [comment]
Closing this for now, until we have an instance we can do the updates to the new architecture

**Pull request closed without merging**